controllers/RecruiterConroller.js

```diff
@@ -0,0 +1,114 @@
+const { body, validationResult, sanitizeBody } = require("express-validator");
+const apiResponse = require("../helpers/apiResponse");
+const { constants } = require("../helpers/constants");
+const auth = require("../middlewares/jwt");
+const RecruiterModel = require("../models/RecruiterModel");
+const UserModel = require("../models/UserModel");
+
+exports.Recruiter = [
+  auth,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
	auth,
```